alvin/script.js

```diff
@@ -0,0 +1,241 @@
+
+
+// https://www.programiz.com/javascript/examples/get-random-item
+// get item from array
+
+function randomWord(words) {
+    const random = Math.floor(Math.random() * words.length);
+
+    const item = words[random];
+
+    return item;
```

Comment:
```suggestion
    const word = words[random];

    return word;
```

Since the function is about word, maybe name your variable as word.